FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+ErrorHandling.swift

```diff
@@ -86,26 +80,39 @@ extension ComposeViewController {
         }
     }
 
-    private func showPlainMessageConfirmationAlert() {
-        showAlertWithAction(
+    private func processSendMessageWithNoValidKeys(error: Error) {
+        let alert = UIAlertController(
             title: "compose_message_encryption".localized,
-            message: "compose_plain_message_confirmation".localized,
-            actionButtonTitle: "compose_send_unencrypted".localized,
-            actionAccessibilityIdentifier: "aid-compose-send-plain",
-            onAction: { [weak self] _ in self?.handleSendTap(shouldSendPlainMessage: true) }
+            message: error.errorMessage,
+            preferredStyle: .alert
         )
-    }
-
-    private func showPlainMessageAlert() {
-        showAlertWithAction(
-            title: "compose_message_encryption".localized,
-            message: "compose_plain_message_alert".localized,
-            cancelButtonTitle: "compose_add_message_password".localized,
-            actionButtonTitle: "compose_send_unencrypted".localized,
-            actionAccessibilityIdentifier: "aid-compose-send-plain",
-            onAction: { [weak self] _ in self?.handleSendTap(shouldSendPlainMessage: true) },
-            onCancel: { [weak self] _ in self?.setMessagePassword() }
+        let sendUnEncryptedAction = UIAlertAction(
+            title: "compose_send_unencrypted".localized,
+            style: .default,
+            handler: { [weak self] _ in
+                self?.handleSendTap(shouldSendPlainMessage: true)
+            }
+        )
+        sendUnEncryptedAction.accessibilityIdentifier = "aid-compose-send-plain"
+        let sendPasswordProtectedAction = UIAlertAction(
+            title: "compose_add_message_password".localized,
+            style: .default,
+            handler: { [weak self] _ in
+                self?.setMessagePassword()
+            }
         )
+        sendPasswordProtectedAction.accessibilityIdentifier = "aid-compose-send-message-password"
+        let cancelAction = UIAlertAction(
+            title: "cancel".localized,
+            style: .cancel
+        )
+        cancelAction.accessibilityIdentifier = "aid-cancel-button"
+        alert.addAction(sendUnEncryptedAction)
```

Comment:
Previously we didn't allow to send unencrypted message for enterprise builds, let's add this restriction back - so users of enterprise build will be able to send only password-protected messages, but not unencrypted

Reply:
Aha, I see